File: 8 kyu/string_cleaning.js
// link : https://www.codewars.com/kata/57e1e61ba396b3727c000251/train/javascript

function stringClean(s){

    let cleanedString ="";
    for (let i=0; i<s.length; i++) {
      if (s[i].charCodeAt(0) >= 48 && s[i].charCodeAt(0) <= 57) {
        
      } else {
        cleanedString += s[i];
      }
    
  
  }
      return cleanedString;
    }

File: 8 kyu/string_cleaning.py
# link : https://www.codewars.com/kata/57e1e61ba396b3727c000251/train/python

def string_clean(s):
    cleaned_string = ""
    for char in s:
        if ord(char) >= 48 and ord(char) <= 57:
            # Skip numeric characters (ASCII 48-57)
            pass
        else:
            cleaned_string += char
    return cleaned_string
